src/Components/ListItems/Item.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Card, Typography, Button } from "material-ui";
+import { CardMedia, CardContent, CardActions } from "material-ui/Card";
+import { withStyles } from "material-ui/styles";
+
+const styles = {
+  card: {
+    margin: 10
+  },
+  media: {
+    height: 200
+  }
+};
+
+const Item = ({ classes, item }) => (
+  <Card className={classes.card} raised={true}>
+    <CardMedia
+      className={classes.media}
+      //image="https://images.pexels.com/photos/574069/pexels-photo-574069.jpeg?w=940&h=650&auto=compress&cs=tinysrgb"
```

Comment:
When done, delete comments and console.log

Reply:
I think I didn't check my code. It worked so it when't strait to github (sorry)